### DISCUSSION TITLE
failed to set upstream: discovery eureka is uninitialized

### DISCUSSION THREAD

**Discussion by ouyangli** [apache/apisix#6917] [Q&A] [answered]
I'm using Eureka for service discovery, but I have a problem。 The error message is as follows：

2022/04/22 11:13:02 [error] 48#48: *19777 [lua] init.lua:512: http_access_phase(): **failed to set upstream: discovery eureka is uninitialized**, client: 192.168.3.109, server: _, request: "GET /v1.0/accessToken HTTP/1.1", host: "192.168.3.214:9080"

I use the following method to install apisix: 
![image](https://user-images.githubusercontent.com/8622409/164960210-1417beb8-22ca-456d-b5a4-81fb4e0ba91f.png)

The config.yaml configuration file of apisix is as follows:
![image](https://user-images.githubusercontent.com/8622409/164965529-cd7cff1b-dcf4-4bb5-b862-f1ae901c7092.png)

eureka is running normally, the screenshot is as follows
![image](https://user-images.githubusercontent.com/8622409/164960348-36d26a0e-df10-4647-a2a4-f1a3c4636ebe.png)

The routing configuration is as follows:
![image](https://user-images.githubusercontent.com/8622409/164965588-7e38b070-5f18-4113-bba5-8bb66b4a66b7.png)

**Comment by tokers** [comment, marked as answer]
What about the APISIX error logs? Could you check it out especially for log entries with error level.

**Reply by ouyangli** [reply]
Solution: Adjust the log level to info, and find the problem

**Comment by cdalexndr** [comment]
@ouyangli what was the issue in your case?

**Comment by talee184** [comment]
@ouyangli what was the issue in your case?
